### DISCUSSION TITLE
Project status and roadmap

### DISCUSSION THREAD

**Discussion by ciosbel** [ThinkUniform/uniformcss#41] [General]
Hello there,
I'm looking around for a minimalist utility-first SASS-based framework to start developing an internal design system for a company, and I stumbled upon your project. I need something flexible and "in-control" without relying on something bigger like bootstrap or tailwind or foundation and such. Ideally, I wanted something like gorko, but unfortunately is not mantained/supported since years.

I might say that I'm quite impressed by this one. I've just playing around with few samples and configs and it looks like I've found what I was looking for. 

So the first thing to say, is just thank you.

On the other hand I haven't found this project as easily as I would. I think it deserves more visibility out there. Looking around on discussions, issues, the web and such, this project seems quite "stale", so to say. Of course it does its job perfectly, but still.
What's the current status? Can be considered prod-ready? Is it still mantained? Is there some ongoing work on it? I've read some 2.x maybe?

Thanks in advance, and thanks again for the work.

**Comment by conormuirhead** [comment]
I can't speak for @jinsupark's plans at all, but I will tell you that we've been using Uniform at Rewatch on both our web app and marketing site for a couple years now, and it's been rock solid. For the occasional area that Uniform doesn't already cover what we need, it's been easy to extend it with additional utilities or to just write a touch of custom utility-style CSS to augment it.
